### PR TITLE
Expose alternate graph return value on ingest

### PR DESCRIPTION
**Pull request by johnkerl**
This maintains compatibility with the one-click API. Additionally, it allows the caller to get the graph object itself so they can do `grf.compute()` and `grf.wait()` on it, and then directly obtain resultst from the computation.

The alternative is `dag.server_logs` and `tasks.fetch_result`, which are cumbersome at best for users who have synchronous needs.

This pattern will be user in the collection-mapper logic, which _absolutely must_ provide a usable synchronous entrypoint.

### CODE REVIEW

**Comment by johnkerl** [comment]
[sc-37815]

**Comment by johnkerl** [comment]
@thetorpedodog ping 🙏

File: src/tiledb/cloud/soma/ingest.py [hunk 1]
```diff
@@ -143,6 +59,43 @@ def run_ingest_workflow(
...
+        access_credentials_name=access_credentials_name,
+    )
+    grf.compute()
+    return {
+        "status": "started",
+        "graph_id": str(grf.server_graph_uuid),
+    }
+
+
+def get_ingest_workflow_graph(
```
Comment (thetorpedodog) [inline src/tiledb/cloud/soma/ingest.py:81]: maybe `build_ingest_workflow_graph` would be a better name.

**Review by thetorpedodog — APPROVED** [review]

**Comment by johnkerl** [comment]
Thanks @thetorpedodog !

**Pull request merged**